### PR TITLE
Completed exercise

### PR DESCRIPTION
**Pull request by t-doan**
css-layout-classes

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            <p>
+              The games of the first genre are mostly single-screen
+              platform/action puzzle types, featuring Donkey Kong as the
+              opponent in an industrial construction setting.
+            </p>
+          </article>
+        </div>
+        <div class="col-1">
+          <img src="./images/dk.png" alt="dk" />
+        </div>
```
Comment (shawnkost) [inline css-layout-classes/index.html:68]: Both of these columns shouldn't have the same class name. One of these columns is bigger than the other, so they should have 2 different column classes

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+              200 video games since his creation.
+            </p>
+            <p>
+              Depicted as a short, pudgyItalian plumber who resides in the
+              Mushroom Kingdom, his adventures generally center upon resecuing
+              Princess Peach from the Koopa villain Bowser.
+            </p>
+          </article>
+        </div>
+      </div>
```
Comment (shawnkost) [inline css-layout-classes/index.html:91]: Same note above ^

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="col-2">
+          <img src="./images/yoshi.png" alt="yoshi" />
+        </div>
```
Comment (shawnkost) [inline css-layout-classes/index.html:32]: Better name would probably be `column-third`. You want to name your column classes based off of how much space they will take up

File: css-layout-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        <div class="col-full">
+          <img src="./images/fox.png" alt="fox" />
+        </div>
+        <div class="col-full">
+          <img src="./images/pikachu.png" alt="pikachu" />
+        </div>
+        <div class="col-full">
+          <img src="./images/ness.png" alt="ness" />
+        </div>
```
Comment (shawnkost) [inline css-layout-classes/index.html:46]: `.column-fourth` would probably be better because these are 4 columns and not 1 full column

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Tai! Mostly the ones related to the last two rows. Those should have 2 different column names, the other ones are just suggestions

**Review by t-doan — COMMENTED** [review]
made the changes

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**